### ISSUE TITLE
A11y_AzuredataStudio_Summary_ScreenReader:Voice Over announce table header name twice.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7941]
GitHubTags: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#Voice Over#Mac
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MACOS MOJAVE Version 10.14.6(18G87)
**Additional Details:**
VoceOver
**Steps to Reproduce :**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button.
8.Start Voice over and observe the announcement.
**Actual:**
Voice Over announce table header name twice.
Expected:
Voice Over should not announce table header name twice.
**User Impact:**
Screen reader User will be confuse by  hearing the table name twice.
**Attachment for Reference:**
[7941_A11y_AzureDataStudio_Summary_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3761014/7941_A11y_AzureDataStudio_Summary_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
I don't think this is a problem with ADS, voice over will read the column twice even for this sample table https://www.w3schools.com/html/html_tables.asp
